tests/description/matchcounter_test.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from dfm.description import MatchCounter
+import spacy
+
+
+class TestMatchCounter:
```

Comment:
A few suggestions for more descriptive/covering/documenting specs:

- Can we show what happens when a token occurs more than once?
- What happens with regarding to different cases? `Soldat`, `soldat`?
- What about normalization? `pæn` == `paen` == `pœn`?
- How do we handle symbols? `pæn,` == `pæn`? `torn-yester` == `tornyester`?
- How do we handle newlines? `pæn,` == `pæn`? `torn\nyester` == `tornyester`?
- Do we lemmatize? `krigen` == `krig`?